jenkins/test-image/README.md

```diff
@@ -0,0 +1,24 @@
+# TestImages
+
+From Kubernetes 1.4, we are making one test-images per Kubernetes version.
+
+
+# Naming:
+User can add more Dockerfile and name it to Dockerfile-[TAG].
+
+[TAG] will be used as part of the built image tag.
+
+
+User can build one more all images,
+for example, make push-single 1.4 will build Dockerfile-1.4 and tag it and push it to k8s-testimages.
```

Comment:
maybe put "make push-single 1.4" in backticks?

same for "make help" on next line and all of the "make foo" commands under usage.